tensorflow/core/ops/state_ops.cc

```diff
@@ -182,8 +182,8 @@ This operation computes
 This operation outputs `ref` after the update is done.
 This makes it easier to chain operations that need to use the reset value.
 
-If `indices` contains duplicate entries, lexicographically later entries
-override earlier entries.
+If `indices` contain multiple entries that reference the same location, any
+entry could be used to do the update; the choice is not defined.
```

Comment:
maybe: 

If `indices` contains multiple entries that reference the same location in `updates`, the output could be updated to any one of the referenced values. Which value is used is undefined.